draft-pda-protocol.md

```diff
@@ -508,14 +508,20 @@ the leader's endpoint URL. The payload is structured as follows:
 struct {
   PDATaskID task_id;
   Time time;
+  Extension extensions<4..2^16-1>;
   PDAEncryptedInputShare encrypted_input_shares<1..2^16-1>;
 } PDAUploadReq;
 ~~~
 
+[OPEN ISSUE: if we include extensions in the AAD then these extensions are required to be
+echoed in each aggregate subrequest. We could either not include this in the AAD, or replace
+the list with a hash of the list (that's sent separately), or something else.]
```

Comment:
In fact, when the extension is used for the purpose of attestation, it probably *needs* to be echoed in each aggregate sub-request.

Reply:
That is where I lean, too, but it does mean extension bloat in each request. That's probably fine, though.